src/modules/storage/BlockStorage.ts

```diff
@@ -43,11 +43,13 @@ export class BlockStorage extends Storages
         var sql =
         `CREATE TABLE IF NOT EXISTS blocks
         (
-            height INTEGER PRIMARY KEY,
-            prev_block TEXT,
-            validators TEXT,
-            merkle_root TEXT,
-            signature TEXT
+            height INTEGER NOT NULL PRIMARY KEY,
+            prev_block TEXT NOT NULL,
+            validators TEXT NOT NULL,
+            merkle_root TEXT NOT NULL,
+            signature TEXT,
+            tx_count INTEGER NOT NULL,
+            enrollment_count INTEGER NOT NULL DEFAULT 0
```

Comment:
Remove the default. Because our query will always insert a number, and the query is only done from one place. Hence default does not make sense.

Reply:
This requires changing the table schema design first.

Reply:
Then let's do that. It's a very simple change.